python/ray/data/block.py

```diff
@@ -1,32 +1,35 @@
-from dataclasses import dataclass
+import resource
```

Comment:
@stephanie-wang looks like this is causing Windows tests to fail!

Reply:
Sorry about that :( I forgot to remove the tests-ok label!